pkg/scheduler/framework/runtime/framework.go

```diff
@@ -390,7 +390,7 @@ func fillEventToPluginMap(p framework.Plugin, eventToPlugins map[framework.Clust
 	// We treat it as: the plugin is not interested in any event, and hence pod failed by that plugin
 	// cannot be moved by any regular cluster event.
 	if len(events) == 0 {
-		klog.Warningf("it's rare have a plugin implemented EnqueueExtensions but returns nil ", "plugin", p.Name())
+		klog.InfoS("it's rare to have a plugin implemented EnqueueExtensions but returns nil ", "plugin", p.Name())
```

Comment:
I'm not sure we should be "judging" an implementation in a log message. Let's just state the fact:

"plugin's EventsToRegister returned nil"

Reply:
SG. Updating the wordings to be neutral.